README.md

```diff
@@ -105,16 +104,15 @@ Both programs are compiled when running "make" in the base directory. If "useLap
   - ` bool isContiguous() `
 ### Element Access
 - **operator()**: returns element at the given coordinates
-  - ` Type operator(size_t) `
-  - ` Type operator()(size_t, size_t) `
-- **roi** specifies a region of interest and returns a submatrix of a given shape. -1 signifies "to the beginning/end of the dimension"
-  - ` Mat& roi(int dim1Start = -1, int dim1End = -1, int dim2Start = -1, int dim2End = -1) `
+  - ` Type operator(size_t...) `
+- **roi** specifies a region of interest and returns a submatrix of a given shape. Parameters are passed in pairs corresponding to each dimension and -1 signifies "to the beginning/end of the dimension". 
+  - ` Mat& roi(int...) `
 - **i** takes either a boolean mask or list of indices and returns a matrix containing only the specified elements
-  - ` Mat<Type> i(Mat<bool> mask)
-  - ` Mat<Type> i(Mat<IntegralType> indices)
+  - ` Mat<Type> i(Mat<bool> mask)`
+  - ` Mat<Type> i(Mat<IntegralType> indices)`
 - **ito** performs the same function as "i" but with an output parameter.
-  - ` Mat<Type> i(Mat<bool> mask, Mat<Type> out) //If the output matrix is too large, a region of interest will be stored.
-  - ` Mat<Type> i(Mat<IntegralType> indices, Mat<Type> out)
+  - ` Mat<Type> ito(Mat<bool> mask, Mat<Type> out) //If the output matrix is too large, a region of interest will be stored.`
```

Comment:
//If the output matrix is too large, the returned matrix will be an roi within out